web_site/domain/user.py

```diff
@@ -0,0 +1,90 @@
+##############################################################################
+### user.py - User model                                                   ###
+##############################################################################
+#                                                                            #
+#   Description : Defines users of the application                           #
+#                                                                            #
+#   External Calls:                                                          #
+#                                                                            #
+#                                                                            #
+#  History                                                                   #
+#                                                                            #
+#  Name       Date               Update                                      #
+# =======    ==========         ========================================     #
+# Jerry      08/21/2016         Module created                               #
+#                                                                            #
+##############################################################################
+
+
+####################
+## IMPORT SECTION ##
+####################
+
+import sys
+
+
+######################
+## GLOBAL VARIABLES ##
+######################
+first_name=''
+last_name=''
+address=''
+user_name=''
+city=''
+state=''
+zip_code=''
+role=''
+
+
+
+################
+## FUNCTIONS  ##
+################
+
+@get_role
+def get_role(arg):
+
+@get_fname
+def get_first_name(arg):
+
+@get_lname
+def get_last_name(arg):
+
+@get_address
+def get_address(arg):
+
+@get_id
+def get_user_name(arg):
+
+@get_city
+def get_city(arg):
+
+@get_state
+def get_state(arg):
+
+@get_zip
+def get_zip_code(arg):
+
+@update_fname
+def set_first_name(arg):
+
+@update_lname
+def set_last_name(arg):
+
+@update_address
+def set_address(arg):
+
+@update_id
+def set_user_name(arg):
+
+@update_city
+def set_city(arg):
+
+@update_state
+def set_state(arg):
+
+@update_zip_code
+def set_zip_code(arg):
+
+@update_role
+def set_role(arg):
```

Comment:
@rolltide0 I'm not really sure what you're going for here. I was envisioning something more like...

```
class User
    def __init__(self):
        self.first_name = None
        self.last_name = None
        self.address = None
        self.user_name = None
        self.city = None
        self.state = None
        self.zip_code = None
        self.role = None
```

Now, one could argue that city, state, zip, etc. are not part of a user. Those would be fields for an `Address` entity but I'll leave that up to the community.